### ISSUE TITLE
Possibility of adding a .is-completed class to the button and table-responsive wrapper for tables

### ISSUE THREAD

**Issue by mauriciomateus** [jgthms/bulma#939]
Hi. I just want to suggest the addition of a .is-completed class to the buttons, which would replace the great spinner after we do, for example, an successful Ajax validation.

Also, In phones, the right sight of wide tables is not accessible. I looked for something like bootstrap's .table-responsive wrapper, but could not find it, so I stole that from bootstrap in my project.

Thanks for creating this great framework.

**Comment by joshmanders** [comment]
> Hi. I just want to suggest the addition of a .is-completed class to the buttons, which would replace the great spinner after we do, for example, an successful Ajax validation.

Why not remove the `is-loading` class instead of tacking on another `is-completed` class?

**Comment by jgthms** [comment]
I agree with @joshmanders suggestion of removing the `is-loading` class? And maybe use the green `is-success` class.

For the table responsive wrapper, I can definitely add one.

**Comment by connectkushal** [comment]
+1 for table responsive wrapper.
